check/check.go

```diff
@@ -589,8 +589,7 @@ resLoop:
 			continue
 		}
 		c.debug("%s\n", par.String())
-		switch par.Object().Name() {
-		case "", "_": // unnamed
+		if name := par.Object().Name(); len(name) < 1 || name[0] == '_' { // unnamed
```

Comment:
nit: use `name == ""` on the left side, as that's more consistent with existing code

Reply:
also update the comment, as `// unnamed` is no longer correct :) how about `// no name or underscore name`?